core/metrics/src/lib.rs

```diff
@@ -41,12 +41,15 @@
 //!
```

Comment:
This `lazy_static!` section should be removed now since the statics are no longer used. And we no longer need `extern crate lazy_static`.

Reply:
@mina86 Yes, I submitted. A week ago, still no reviews :/

https://github.com/near/nearcore/pull/5318
https://github.com/near/nearcore/pull/5366
https://github.com/near/nearcore/pull/5374